modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( \Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio $widget ): void {
+		$settings = $widget->get_settings_for_display();
+
+		/**
+		 * Map args to internals
+		 */
+
+		//  Identity Image
+		$identity_image_props_style = $settings['identity_image_style'] ?? 'profile';
+		$identity_image_value       = $settings['identity_image'] ?? [];
+
+		//  Bio Heading
+		$bio_heading_output    = '';
```

Comment:
Also in other places.

```suggestion
		$bio_heading_output = '';
```